templates/articles/record_article_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load sections_tags %}
+{% load wagtailmetadata_tags %}
+{% block meta_tag %}
+  {% meta_tags %}
+{% endblock %}
+
+{% block content %}
+{% endblock %}
+
+{% block extra_js %}
+ {% endblock %}
```

Comment:
Nitpick: preferbly we'd write the endblock as `{% endblock block_name %}` to explicitly close rather than implicitly close.